### DISCUSSION TITLE
control file stream is not good

### DISCUSSION THREAD

**Discussion by wkitlasten** [usgs/pestpp#275] [General]
Can anyone provide more information on the following? Not sure where to start looking for this week's goof!

Error condition prevents further execution: 
control file stream is not good

**Comment by jtwhite79** [comment]
usually it means the control file wasnt found or that its not formatted properly.  Is the control file where you told pest++ it should be?  does pest++ report that its read parts of the file successfully before that message appears?

**Comment by wkitlasten** [comment]
Sorry, I should have included more of the output. It is finding the control file:

```
noptmax:0, npar_adj:5883, nnz_obs:12


             pestpp-ies: a GLM iterative ensemble smoother

                   by the PEST++ development team


version: 5.2.7
binary compiled on Sep  4 2023 at 12:59:48

started at 11/10/23 15:07:11
...processing command line: ' pestpp-ies.exe test.pst'
...using serial run manager

using control file: "test.pst"
in directory: "C:\Users\wesk\Documents\GW_projects\test"
on host: "derpityderpderp"

processing control file test.pst
Error condition prevents further execution: 
control file stream is not good
```

The rec file gets to:
```
version: 5.2.7
binary compiled on Sep  4 2023 at 12:59:48

using control file: "test.pst"
in directory: "C:\Users\wesk\Documents\GW_projects\test"
on host: "derpityderpderp"
started at 11/10/23 15:07:11
```

**Comment by jtwhite79** [comment]
The "in directory" listings are difference between sceen output and rec file...one has a space in the name.  Also is test.pst valid?  Can pyemu read it?

**Reply by cnicol-gwlogic** [reply]
And the hosts differ too

**Comment by wkitlasten** [comment]
Ha... feeble attempt at anonymity, sorry. I just checked and they are the same. I had the wrong `cwd` argument in pyemu.os_utils.run() and neglected to take a step back from my script... doh. So yes, that error arises when pestpp can't read the control file! Thanks.
